src/style.css

```diff
@@ -1,3 +1,96 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
It's better to reset margins and paddings only for the elements that you actually need instead of all